frontend/src/components/applications/EligibilityNotificationDialog.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <AppDialog v-model="isDisplayed" title="Eligibility Notification" :is-loading="isLoading" persistent max-width="55%" @close="closeDialog">
+    <template #content>
+      <div class="confirm-dialog-text">
+        Thank you for your interest in joining the $10 a Day ChildCareBC Program. Based on the answers you have provided, your facility has not met the minimum eligibility criteria to participate at
+        this time. For more information about the $10 a Day ChildCareBC Program, please visit our
+        <a href="https://www2.gov.bc.ca/gov/content?id=F0D0FD2A00064CDDBF634CF83E4E2599" target="_blank">website</a>
+      </div>
+    </template>
+    <template #button>
+      <v-row justify="space-around">
+        <v-col cols="12" md="6" class="d-flex justify-center">
+          <AppButton id="dialog-cancel-application" :primary="false" size="large" min-width="250px" max-width="450px" :loading="isLoading" @click="cancel">Cancel Application</AppButton>
```

Comment:
After clicking No on one of the knockout questions - I saw the popup then clicked 'cancel' on the popup, as prompted. However - after clicking cancel, I was still able to close the popup, change my answers all to yes - and then proceed to the end of the application, where I could not submit (assuming because there is no application left)? But I could see how that would maybe be confusing for the user. 
 
Just thinking maybe we should do something after a user clicks cancel - send them back to the home page maybe?

Reply:
good point! I will update it

Reply:
it's updated. thanks Jen!